Detectors/PHOS/simulation/src/Detector.cxx

```diff
@@ -142,7 +142,7 @@ Bool_t Detector::ProcessHits(FairVolume* v)
   //    return false ; //  We are not inside a PBWO crystal
 
   Int_t moduleNumber;
-  fMC->CurrentVolOffID(10, moduleNumber); // get the PHOS module number ;
+  fMC->CurrentVolOffID(11, moduleNumber); // get the PHOS module number ;
```

Comment:
please make a named constant for this number and document what it is

Reply:
This constant is used in one place and is used in exceptional cases, if one adds one more embedded volume. I added description but there is no need in constant

Reply:
It makes still sense to document the meaning of 11? Is it `11` levels up the volume hierarchy or something?

Reply:
Exactly: this is offset btw. active volume and parent module to get module number. I added a comment.